website/docs/extensions/documentation-generation.md

```diff
@@ -0,0 +1,52 @@
+# Generating Documentation
+
+`scarb doc` is a tool that generates documentation based on the code comments. Generation supports different out formats. The result is being placed inside the `/target/doc`.
+
+## Supported output formats
+
+- Markdown. Fully supported by [mdBook](https://rust-lang.github.io/mdBook/). (Default)
+- Custom JSON
+
+## Available type of comments
+
+As for now, only the `///` comment prefix is supported.
+
+## mdBook
+
+Generated markdown can be used to create documentation book.
+Requirements:
+
+- Install [mdBook](https://rust-lang.github.io/mdBook/guide/installation.html) running `cargo install mdbook`.
+- Run `scarb doc` inside the project root.
+- Run `mdbook build` (or `mdbook serve`) inside the generated documentation target (`/target/doc/<PACKAGE-NAME>`).
+
+## Examples
+
+Let's take for a example simple Cairo project initalized using `scarb new`. The code inside `lib.cairo`:
```

Comment:
> Let's take, as an example, a simple Cairo project initialized using scarb new.

This indicates, that if I run `scarb new` I will get a snippet below? Which is not true afaik.